### DISCUSSION TITLE
Feature request: add support for lvgl library

### DISCUSSION THREAD

**Discussion by jumperrg** [sbrin/lopaka#173] [Ideas]
Hello, thank you for the amazing tool! Do you think it is possible to add such library support? 
The library is great because you can make UI in a way like html - using flex boxes, alignment, margins, borders, etc.
If it's possible I could contribute to the code.

**Comment by sbrin** [comment]
That's a great idea. I have it on my mind since 2023 🤓
The problem is LVGL API and it's complexity. I have not enough experience in embedded programming to figure out all nuances. Maybe we can cooperate on that?

We can start with something simple like display a text and go step by step. What do you think?

**Reply by jumperrg** [reply]
Nice, I would love to cooperate!
"We can start with something simple like display a text and go step by step. What do you think?" - I think it's a great idea! 
I guess I should try to run the project locally and get familiar with it at first. 
Can you give me any hints?
